root/etc/supervisor.conf

```diff
@@ -15,14 +15,14 @@ daemon=off
 priority = 2
 
 [program:webapp]
-environment=NODE_ENV="production",PORT=3000
+environment=NODE_ENV="production",PORT="%(ENV_WEB_APP_PORT)s",HTTPS_PROXY="%(ENV_HTTPS_PROXY)s",HTTP_PROXY="%(ENV_HTTP_PROXY)s",NO_PROXY="%(ENV_NO_PROXY)s"
```

Comment:
Here the port for webapp is made configurable as well as the proxy settings is being passed to `webapp` context.